op-deployer/pkg/deployer/state/intent.go

```diff
@@ -300,19 +311,19 @@ func NewIntentCustom(l1ChainId uint64, l2ChainIds []common.Hash) (Intent, error)
 }
 
 func NewIntentStandard(l1ChainId uint64, l2ChainIds []common.Hash) (Intent, error) {
+	opcmAddr, err := standard.ManagerImplementationAddrFor(l1ChainId, artifacts.DefaultL1ContractsLocator.Tag)
```

Comment:
Can we rename `ManagerImplementationAddrFor` to `OpcmImplAddrfor` to make it consistent with standard contract names?